rest.getinsomnia.Insomnia.desktop

```diff
@@ -0,0 +1,7 @@
+[Desktop Entry]
```

Comment:
You should bring it upstream.

Reply:
OK. So you advice to pull request on beginning to Insomnia repository and provide new directory, for example *flathub* with two additional files: *rest.getinsomnia.Insomnia.appdata.xml* and *rest.getinsomnia.Insomnia.desktop*.

After that we should upgrade this manifest to download this data from upstream repository. That's right?

Reply:
It's ok to just submit a pull request there for now so it doesn't get lost. Probably `xdg` or `appdata` would be a better name.